src/functions/services/logging/logger.test.ts

```diff
@@ -0,0 +1,125 @@
+import { LogAttributes } from "@aws-lambda-powertools/logger/lib/cjs/types/Log";
+import { ILoggerAdapter, Logger } from "./logger";
+import { LogMessage, RegisteredLogMessages } from "./types";
+import { Context } from "aws-lambda";
+
+describe("Logger", () => {
+  describe("Given there is a message to log", () => {
+    it("Writes a log with registered log data ", () => {
+      const loggingAdapter = new MockLoggingAdapter();
+      const mockLogger = new Logger<MockMessage>(
```

Comment:
Tiny tiny nut but why is this const `mockLogger`? Isn't this the real logger so we could call it logger? No value provided here so happy if you want to dismiss. If you want to change then there's other occurrences, I'm just going to comment once

Reply:
I think this is a good shout. Updated